ReactAndroid/src/main/java/com/facebook/react/ReactInstanceManager.java

```diff
@@ -88,6 +88,11 @@
   public abstract void onBackPressed();
 
   /**
+   * This method will give JS the opportunity to receive intents via Linking.
+   */
+  public abstract void onNewIntent(Intent intent);
```

Comment:
@andreicoman11 Should we add this to https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/bridge/ActivityEventListener.java so that all modules can use it?